src/rev_ai/sentiment_analysis_client.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""Client used or interacting with out sentiment analysis api"""
+
+from .generic_api_client import GenericApiClient
+from .models import SentimentAnalysisJob, SentimentAnalysisResult
+
+
+class SentimentAnalysisClient(GenericApiClient):
+    """Client for interacting with the Rev AI sentiment analysis api"""
+
+    # Default version of Rev AI sentiment analysis api
+    api_version = 'v1'
+
+    # Default api name of Rev AI sentiment analysis api
+    api_name = 'sentiment_analysis'
+
+    def __init__(self, access_token):
+        """Constructor
+
+        :param access_token: access token which authorizes all requests and links them to your
+                             account. Generated on the settings page of your account dashboard
+                             on Rev AI.
+        """
+
+        GenericApiClient.__init__(self, access_token, self.api_name, self.api_version,
+                                  SentimentAnalysisJob.from_json, SentimentAnalysisResult.from_json)
+
+    def submit_job_from_text(self,
+                             text=None,
+                             metadata=None,
+                             callback_url=None,
+                             delete_after_seconds=None,
+                             language=None):
```

Comment:
Now that I merged AISDK-190 this should have notification_config as in the other features